check_img_lazy.py

```diff
@@ -0,0 +1,61 @@
+import os
+import sys
+import subprocess
+from bs4 import BeautifulSoup
```

Comment:
Is it a possibility to use standard Python libraries for parsing HTML files?

What's your opinion here @shivamgupta2020 ? We need to instruct people to install some dependencies if we go with BeautifulSoup. However, we might have some non-standard HTML content that might not be parseable with standard Python libraries.

Let's ask @knative/productivity-leads and @cardil specifically.

Reply:
@aliok, I experimented with HTMLParser and IXML for parsing HTML files, but encountered numerous errors and incomplete parsing. Therefore, I switched to BeautifulSoup. I also think we should gather opinions from others on this approach.

Reply:
cc @pierDipi @cardil